bHY6yh-d10089/cod-bxxqqq_bHY6yh-d10089.xml

```diff
@@ -0,0 +1,290 @@
+          <lb ed="#B"/><quote xml:id="bHY6yh-d10089-Qd1e137">Non proderunt diuitie ultionis 
+          <lb ed="#B"/>Iustitia autem liberabit a morte</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e146">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d10089-Rd1e147">MAlachias propheta dies</ref> <quote xml:id="bHY6yh-d10089-Qd1e150">veniet suc<lb ed="#B" break="no"/>censsus
+          quasi caminus et erunt superbi 
+          <lb ed="#B"/>et omnes facientes impietatem sti<lb ed="#B" break="no"/>pula 
+          et inflamabit eos dies ve<lb ed="#B" break="no"/>niens</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e158">
+          <ref xml:id="bHY6yh-d10089-Rd1e161">MAtheus in ewangelo capitulo 12</ref> <quote xml:id="bHY6yh-d10089-Qd1e162">Omne 
+          <lb ed="#B"/>verbum otiosum quod locuti fuerint ho<lb ed="#B" break="no"/>mines 
+          reddent <unclear>romem</unclear> de <unclear>oo</unclear> eo in die 
+          <lb ed="#B"/>iudicii ex verbis enim tuis iustifica<lb ed="#B" break="no"/>beris
+          et ex verbis tuis <unclear>condempnaberis</unclear></quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e172">
+          <lb ed="#B"/>¶ <ref xml:id="bHY6yh-d10089-Rd1e175">capitulo 13</ref> <quote xml:id="bHY6yh-d10089-Qd1e174">In consummatione seculi mit<lb ed="#B" break="no"/>tet
+          filius hominis angelos suos et col<lb ed="#B" break="no"/>ligent
+          de regno suo omnia scanda<lb ed="#B" break="no"/>la 
+          et omnes qui faciunt iniquita<lb ed="#B" break="no"/>tem 
+          et mittet eos in caminum ig<lb ed="#B" break="no"/>nis 
+          ibi erit fletus et stridor den<lb ed="#B" break="no"/>tium 
+          tunc iusti fulgebunt sicud sol
+          <lb ed="#B"/>in regno patris eorum</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e193">
+          ¶ <quote xml:id="bHY6yh-d10089-Qd1e195">In consummatione 
+          <lb ed="#B"/>seculi exibunt angeli et separabunt 
+          <lb ed="#B"/>malos de medio iustorum et / mittent 
+          <lb ed="#B"/>eos in caminum ignis ardentis ibi erit
+          <lb ed="#B"/>fletus et stridor deutium</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e207">
+          ¶ <ref>capitulo 22</ref> <quote xml:id="bHY6yh-d10089-Qd1e209">In 
+          <lb ed="#B"/>resurreccione nec nubent ne nubentur 
+          <lb ed="#B"/>sed sunt sicud angeli in caelo</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e215">
+          ¶ <ref>capitulo 24</ref> <quote xml:id="bHY6yh-d10089-Qd1e217">Ac<lb ed="#B" break="no"/>ceserunt
+          ad ihesum discipuli eius secreto 
+          <lb ed="#B"/>dicentes dic nobis signum aduentus et 
+          <lb ed="#B"/>consumationis seculi et respondens ihesus
+          <lb ed="#B"/>dixit eis videte nequis nos se ducat 
+        <cb ed="#B" n="b"/>
+          <lb ed="#B"/>Multi enim venient in nomine dicen<lb ed="#B" break="no"/>tes 
```

Comment:
no need to change this one, but in the future try to keep your `<cb/>` and `<pb/>` tags flush with the `<lb/>`

```suggestion
          <lb ed="#B"/>dixit eis videte nequis nos se ducat 
          <cb ed="#B" n="b"/>
          <lb ed="#B"/>Multi enim venient in nomine dicen<lb ed="#B" break="no"/>tes 
```